env0/data_project_test.go

```diff
@@ -17,6 +17,15 @@ func TestProjectDataSource(t *testing.T) {
 		Description: "A project's description",
 	}
 
+	archivedProject := client.Project{
+		Id:          "id0",
+		Name:        "my-project-1",
+		CreatedBy:   "env0",
+		Role:        "role0",
+		Description: "A project's description",
+		IsArchived:  true,
+	}
```

Comment:
please use the values from `project` where only `Id` and `IsArchived` should be different

Reply:
done.

Reply:
please comment with the commit hash

Reply:
4c9a58c88842bf9b52c5b8683cd5a713b54d602f

Reply:
58e077d89f144f056ed038ff261bb52cf10ce0e5